main.go

```diff
@@ -339,6 +344,13 @@ func addIdentifiers(spelling *hunspell.Spell, pkgs []*packages.Package) error {
 		for _, f := range p.Syntax {
 			ast.Walk(v, f)
 		}
+		for _, dep := range p.Imports {
```

Comment:
nit: I think you want Deps, as that includes indirect imports too.

Reply:
Sorry, where is that? I don't see a Deps field, but this will recur up the tree.

Reply:
Ah, my bad; it's a field in `go list -json`, but it's omitted from `go/packages`. Imports sounds good if you're going to recurse yourself.